packages/vsce/package.json

```
@@ -417,6 +418,15 @@
         "command": "cics-extension-for-zowe.purgeTask",
         "title": "Purge Task",
         "category": "IBM CICS for Zowe Explorer"
+      },
+      {
+        "command": "cics-extension-for-zowe.toggleResourceSetting",
+        "title": "Click to customize visible CICS Resources",
```

I don't think we tend to put "Click to..." on each button's tooltip.
```suggestion
        "title": "Toggle visible CICS Resources",
```